34.md

```diff
@@ -17,48 +17,60 @@ Git repositories are hosted in Git-enabled servers, but their existence can be a
   "kind": 30617,
   "content": "",
   "tags": [
-    ["d", "<repo-id>"],
+    ["d", "<repo-id>"], // usually kebab-case short name
     ["name", "<human-readable project name>"],
     ["description", "brief human-readable project description>"],
     ["web", "<url for browsing>", ...], // a webpage url, if the git server being used provides such a thing
     ["clone", "<url for git-cloning>", ...], // a url to be given to `git clone` so anyone can clone it
     ["relays", "<relay-url>", ...] // relays that this repository will monitor for patches and issues
+    ["earliest-unique-commit", "<commit-id>"] // usually root commit but a recent commit for forks
+    ["r", "<earliest-unique-commit-id>"] // so clients can subscribe to all events related to a local git repo
+    ["maintainers", "<other-recognized-maintainer>", ...]
   ]
 }
 ```
 
-The tags `web`, `clone`, `relays` can have multiple values.
+The tags `web`, `clone`, `relays`, `maintainers` can have multiple values.
 
 Except `d`, all tags are optional.
 
 ## Patches
 
 Patches can be sent by anyone to any repository. Patches to a specific repository SHOULD be sent to the relays specified in that repository's announcement event's `"relays"` tag. Patch events SHOULD include an `a` tag pointing to that repository's announcement address.
 
+Patches in a patch set SHOULD include an nip10 `e` `reply` tag pointing to the to the previous patch. 
+
+The first patch revision in a patch revision SHOULD include a nip10 `e` `reply` to the oringal root patch.
```

Comment:
```suggestion
The first patch revision in a patch revision SHOULD include a nip10 `e` `reply` to the original root patch.
```

Reply:
I made this updated via an amended commit